### DISCUSSION TITLE
Completions of LSP server truncated

### DISCUSSION THREAD

**Discussion by cangermueller** [hrsh7th/nvim-cmp#2018] [Q&A]
Hey! 

Completions of one of my LSP servers are relative to characters entered so far, whereas completions of other sources (buffer, etc, …) include the characters entered so far in the prefix:
![image](https://github.com/user-attachments/assets/a1506e34-2000-4d6a-9d2d-7dd4836bb594)

When I select/accept the completion of the LSP server, the characters entered so far are replaced:
![image](https://github.com/user-attachments/assets/a2b05d91-a7a1-4f53-bedd-f3c276b43218)

Is there a way to either a) also include the prefix character in completions of the LSP server, or b) append completions to the characters entered so far instead of replacing them?

Thanks!

**Comment by fitrh** [comment]
Can you share your cmp config? Especially the mapping

**Comment by cangermueller** [comment]
Thanks for your response. I found out that it was an issue with my LSP server, which got fixed now.
